File: DSRFQ.Web/Modules/Administration/UserInvitations/UserInvitationsForm.cs
using Serenity.ComponentModel;
using System;

namespace DSRFQ.Administration.Forms;

[FormScript("Administration.UserInvitations")]
[BasedOnRow(typeof(UserInvitationsRow), CheckNames = true)]
public class UserInvitationsForm
{
    public int InvitedByUserId { get; set; }
    public string EmailAddress { get; set; }
    public bool Accepted { get; set; }
}

File: DSRFQ.Web/Modules/ServerTypes/Administration/UserInvitationsForm.ts
import { IntegerEditor, StringEditor, BooleanEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface UserInvitationsForm {
    InvitedByUserId: IntegerEditor;
    EmailAddress: StringEditor;
    Accepted: BooleanEditor;
}

export class UserInvitationsForm extends PrefixedContext {
    static readonly formKey = 'Administration.UserInvitations';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!UserInvitationsForm.init)  {
            UserInvitationsForm.init = true;

            var w0 = IntegerEditor;
            var w1 = StringEditor;
            var w2 = BooleanEditor;

            initFormType(UserInvitationsForm, [
                'InvitedByUserId', w0,
                'EmailAddress', w1,
                'Accepted', w2
            ]);
        }
    }
}